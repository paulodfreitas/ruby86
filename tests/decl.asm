.module entry_code
.pseg
.org 0
        irmovl $3, %edx
        decl %edx
        decl %edx
        decl %edx
        decl %edx
        decl %edx
        halt
.end
