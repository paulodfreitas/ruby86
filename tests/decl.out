@ 0000
30
82
03
00
00
00
65
82
65
82
65
82
65
82
65
82
10
@ 0011
